mirgecom/eos.py

```diff
@@ -122,7 +129,8 @@ def gas_const(self):
         return self._gas_const
 
     def internal_energy(self, cv: ConservedVars):
-        r"""Get internal energy
+        r"""
+        Get internal energy.
```

Comment:
I don't find this docstring helpful.

Reply:
I agree.  I was responding to some of the issues identified by the doc linter.